src/Core/components/Answer.js

```diff
@@ -988,9 +989,15 @@ export default class Answer extends InlineComponent {
             currentResponses.push(...component.stateValues.values)
             componentType.push(...Array(component.stateValues.values.length)
               .fill(ct));
-          } else {
+          } else if (component.stateValues.value !== undefined) {
             currentResponses.push(component.stateValues.value)
             componentType.push(ct)
+          } else if (component.stateValues.formula instanceof me.class) {
```

Comment:
Not worth blocking this bugfix PR, but notating for the future I had a discussion with Duane about if we might want to formalize a concept of declaring a "primary" state variable for a component. For example here I would prefer we not have and instanceof check and instead be able to refer to the "value" state variable for all cases. Although admittedly creating an "alias" for the formula state variable that would be called value would be confusing because it likely would make people think about the value of the function evaluated for a given input.